airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -31,6 +32,8 @@ acceptance_tests:
             bypass_reason: "This stream can't be seeded in our sandbox account"
           - name: "transfers"
             bypass_reason: "This stream can't be seeded in our sandbox account"
+          - name: "products"
+            bypass_reason: "This stream can't be seeded in our sandbox account"
```

Comment:
Why bypassing this stream?

Reply:
Sorry I wasn't sure with the empty_streams. Do we put streams in here when it _could_ be empty? I mean the credentials I tested has no products so the integration test fails if I don't bypassing it here.